Lib/test/test_platform.py

```diff
@@ -741,5 +745,73 @@ def test_parse_os_release(self):
         self.assertEqual(len(info["SPECIALS"]), 5)
 
 
+class CommandLineTest(unittest.TestCase):
+    def setUp(self):
+        platform.invalidate_caches()
+        self.addCleanup(platform.invalidate_caches)
+
+    @staticmethod
+    def text_normalize(string):
+        """Dedent *string* and strip it from its surrounding whitespaces.
+
+        This method is used by the other utility functions so that any
+        string to write or to match against can be freely indented.
+        """
+        return dedent(string).strip()
+
+    def invoke_platform(self, *flags):
+        output = io.StringIO()
+        with contextlib.redirect_stdout(output):
+            platform._main(args=flags)
+        return self.text_normalize(output.getvalue())
```

Comment:
If we don't really care about parsing the output, we can do without using `textwrap.dedent`
```suggestion
    def invoke_platform(self, *args):
        output = io.StringIO()
        with contextlib.redirect_stdout(output):
            platform._main(args)
        return output.getvalue()
```